### PR TITLE
feat/owner assets path

### PR DESCRIPTION
**Pull request by tx0c**
* 3d775fd add `/owner/[address].tsx` path
* c8cbe2a add getRecentLogs in the context

### CODE REVIEW

File: src/components/LogbookContext/index.tsx [hunk 1]
```diff
@@ -77,10 +79,11 @@ type ReducerAction =
     }
 
 type Context = {
+  getRecentLogbooks: (limit?: number) => Promise<void>
   getLogbook: (tokenId: string) => Promise<void>
   logbooks: { [tokenId: string]: Logbook | undefined }
 
-  getOwnNFTs: () => Promise<void>
+  getOwnNFTs: (owner?: string | null | undefined) => Promise<void>
```
Comment (tx0c) [inline src/components/LogbookContext/index.tsx:86]: is @robertu7 aware of an address type?  to combine all `string | null | undefined` types?

File: src/components/LogbookContext/index.tsx [hunk 2]
```diff
@@ -246,9 +249,108 @@ export const LogbookProvider = ({
...
+          },
+        },
+      })
+    } catch (err) {
+      console.error("getRecentLogbooks ERROR:", err)
+    }
+  }
+
+  // retrieve all logbooks own by the given owner, default to current account
+  const getOwnNFTs = async (owner: string | null | undefined = account) => {
```
Comment (tx0c) [inline src/components/LogbookContext/index.tsx:352]: an `address` type to be used here?

**Review by robertu7 — CHANGES_REQUESTED** [review]

File: src/components/LogbookContext/index.tsx [hunk 3]
```diff
@@ -246,9 +249,108 @@ export const LogbookProvider = ({
     }
   }
 
-  // retrieve all logbooks own by current account
-  const getOwnNFTs = async () => {
-    if (!account) {
+  // get recent logbooks at most limit? number
+  const getRecentLogbooks = async (limit: number = 10) => {
```
[the remark this answers is not in the document]
Comment (tx0c) [reply src/components/LogbookContext/index.tsx:253]: yes, this is exported, and can be re-used

[the remark this answers is not in the document]
Comment (tx0c) [reply src/components/LogbookContext/index.tsx:253]: after #213 merged, this can be rebased and will be much cleaner

**Review by robertu7 — APPROVED** [review]

**Pull request merged**